internal/services/batch/batch_pool_data_source.go

```diff
@@ -81,118 +86,240 @@ func dataSourceBatchPool() *pluginsdk.Resource {
 					},
 				},
 			},
-			"storage_image_reference": {
+			"container_configuration": {
```

Comment:
this diff is making it hard to review given the large number of changes, could you undo the reordering here and open a separate PR to change that?

Reply:
I reorder this since the original order did not follow the alphabetic rules, let me see what I can do about this.